addons/resource.language.en_gb/resources/strings.po

```diff
@@ -11003,13 +11012,12 @@ msgctxt "#20448"
 msgid "Could not download information"
 msgstr ""
 
+#: xbmc/video/VideoInfoScanner.cpp
 msgctxt "#20449"
-msgid "Unable to connect to remote server"
+msgid "Unable to connect to remote server. Would you like to continue scanning?"
```

Comment:
I want to have a PR that can be merged without to much fuss. If the rest needs to be taken care of as well some one else is free to do so or I'll maybe do it in future but not now.